code/modules/awaymissions/zlevel.dm

```diff
@@ -3,7 +3,7 @@ proc/createRandomZlevel()
 		return
 
 	var/list/potentialRandomZlevels = list()
-	admin_notice("\red \b Searching for away missions...", R_DEBUG)
+	admin_notice("<span class='warning'>\b Searching for away missions...</span>", R_DEBUG)

```

Comment:
Since we're already looking at these.

`\b` is the same as `<b>...</b>`. So this should actually be `<span class='danger'>` without the `\b`.